pages/addTask.js

```diff
@@ -9,93 +9,107 @@ export default function AddTaskForm({ handleAddTask }) {
   }
 
   return (
-    <StyledForm onSubmit={handleSubmit}>
-      <h1>Add Task</h1>
+    <Wraper>
+      <StyledForm onSubmit={handleSubmit}>
```

Comment:
When refactoring, consider to extract the form as component and use it here as well as where you edit the data.